File: 1470-shuffle-the-array/1470-shuffle-the-array.java
class Solution {
    public int[] shuffle(int[] nums, int n) {
        if(n==0){
            return nums;
        }
       else{
           int arr[]=new int[nums.length];
           int j=0;
           for(int i=0;i<nums.length && j<nums.length;i++){
               arr[j]=nums[i];
               j++;
               arr[j]=nums[i+n];
               j++;
           }
           return arr;
           
       }        
        
    }
}

File: 1470-shuffle-the-array/1470-shuffle-the-array.cpp
class Solution {
public:
    vector<int> shuffle(vector<int>& nums, int n) {
        vector<int> v(2*n, 0);
        int k=0;
        for(int i=0;i<n;i++) {
            v[2*k] = nums[i];
            v[2*k + 1] = nums[n+i];
            k++;
        }
        return v;
    }
};